app/models/timeline_event.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: timeline_events
+#
+#  id                  :bigint           not null, primary key
+#  annotation          :string
+#  creatable_type      :string
+#  event_type          :string           not null
+#  title               :string           not null
+#  created_at          :datetime         not null
+#  updated_at          :datetime         not null
+#  application_form_id :bigint
+#  creatable_id        :integer
+#
+# Indexes
+#
+#  index_timeline_events_on_application_form_id  (application_form_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (application_form_id => application_forms.id)
+#
+class TimelineEvent < ApplicationRecord
+  belongs_to :application_form
+  belongs_to :creatable, polymorphic: true, optional: true
```

Comment:
```suggestion
  belongs_to :creator, polymorphic: true, optional: true
```

Perhaps this would be better, but it does go against the convention of `able` for polymorphic fields. I'd suggest it shouldn't be optional though until we find there's a need for it.